### DISCUSSION TITLE
希望公招能够只在有高星tag时才进行

### DISCUSSION THREAD

**Discussion by luvleizi** [MaaAssistantArknights/MaaAssistantArknights#7976] [Ideas]
设置的招募次数少的话，刷新次数经常溢出，招募次数拉满又导致招募许可快速消耗，很快就没有了，然后还是回到招募次数少的情况，刷新次数容易溢出。无招募许可时继续尝试刷新 tags 的功能会不断地将三四位的格子刷到出现高星tag，但这时招募许可数量不足，只会在一二号位进行公招，三四位长时间处于有高星tag但一直都没有进行公招。希望能在公招策略里加个只在有高星tag时才公招的选项，我可以半个月手动换一下策略。

**Comment by ABA2396** [comment]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/99072975/eaf8de12-1895-4429-ae10-cdbec8fabcb2)

**Reply by luvleizi** [reply]
没看懂 是说3星设置1:00这个吗

**Reply by luvleizi** [reply]
![Untitled](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/107036550/d7f3242c-ee66-4564-8156-2ffa602af513)

**Reply by ABA2396** [reply]
你把自动确认3星去掉不就好了吗
